.reuse/dep5

```diff
@@ -46,3 +46,8 @@ License: CC0-1.0
 Files: doc/img/logo.png
 Copyright: 2023 Yaskawa America, Inc.
 License: CC-BY-NC-ND-4.0
+
+# strtoll implementation
+Files: libmicroros*/libmicroros.*
+Copyright: 2014, The Regents of the University of California.
+License: BSD-3-Clause
```

Comment:
I'm going to remove this, as UC didn't write that function, it's just the organisation that originated the license.

Proper attribution of the other parts of `libmicroros` is something we have on our TODO list already, and singling this out doesn't seem correct.